pkg/apis/core/types_shoot.go

```diff
@@ -154,6 +154,18 @@ type ShootStatus struct {
 	MigrationStartTime *metav1.Time
 	// Credentials contains information about the shoot credentials.
 	Credentials *ShootCredentials
+	// LastMaintenance holds information about the last maintenance operations on the Shoot.
```

Comment:
```suggestion
	// LastMaintenance holds information about the last maintenance operation on the Shoot.
```

Same for other API packages

Reply:
I kept it as operations, cause there can be multiple changes on a shoot, shoot like Kubernetes version, worker machine image, or worker Kubernetes version.